src/main/resources/portal.properties.EXAMPLE

```diff
@@ -106,8 +106,11 @@ ldap.attributes.last_name=sn
 ldap.attributes.given_name=givenName
 ldap.attributes.email=mail
 
+# study view settings
 # always show studies with this group
 always_show_study_group=
+mdacc.heatmap.study.meta.url=http://bioinformatics.mdanderson.org/study2url?studyid=
```

Comment:
cbioportal.org is under HTTP protocol, but we do have some of instances under HTTPS protocol. Directly using the link here may not work. It will run into the cross-domain issue. See screenshot. And we are using iframe here, I don't think server-side proxy will help either.

![screen shot 2016-10-03 at 12 10 01 pm](https://cloud.githubusercontent.com/assets/5400599/19044498/9207c5b8-8962-11e6-828c-6b9c2d6d2b7c.png)